src/posts/models/toTags/hasPostTag.props.ts

```diff
@@ -8,7 +8,11 @@ export class HasPostTagProps implements RelationshipProps {
     @ApiProperty({ type: String })
     tagName: string;
 
+    @ApiProperty({ type: String })
+    tagColor: string;
```

Comment:
I think I made a mistake here. There is no need for the `HasPostTagProps` class to contain the properties of the `PostTag` class. Because the HAS_POST_TAG relationship doesn't need any properties.
So, can you remove all the properties of this `HasPostTagProps` class? (don't remove the whole class tho. we need to have the class at least - just to follow the project's conventions).

Oh, I think this is the same case for PostType. Uh, could you do the same for that please? 😅 I'm trolling i guess.

Reply:
No worries, I'll remove them